pocs/tests/test_observatory.py

```diff
@@ -65,23 +67,6 @@ def test_bad_site(simulator, config):
         Observatory(simulator=simulator, config=conf, ignore_local_config=True)
 
 
-def test_bad_mount_port(config):
-    conf = config.copy()
-    simulator = hardware.get_all_names(without=['mount'])
-    conf['mount']['serial']['port'] = 'foobar'
-    with pytest.raises(error.MountNotFound):
-        Observatory(simulator=simulator, config=conf, ignore_local_config=True)
-
-
-@pytest.mark.without_mount
-def test_bad_mount_driver(config):
-    conf = config.copy()
-    simulator = hardware.get_all_names(without=['mount'])
-    conf['mount']['driver'] = 'foobar'
-    with pytest.raises(error.MountNotFound):
-        Observatory(simulator=simulator, config=conf, ignore_local_config=True)
-
-
 def test_can_observe(config, caplog):
```

Comment:
Should this test be named `test_cannot_observe`? And if so we should say the reason why (e.g. `test_no_mount_cannot_observe`)